src/azure-cli/azure/cli/command_modules/storage/__init__.py

```diff
@@ -23,12 +23,35 @@ def __init__(self, cli_ctx=None):
 
     def load_command_table(self, args):
         super(StorageCommandsLoader, self).load_command_table(args)
-        from azure.cli.command_modules.storage.commands import load_command_table
+        from azure.cli.command_modules.storage.commands_v2 import load_command_table
         load_command_table(self, args)
         return self.command_table
 
     def load_arguments(self, command):
         super(StorageCommandsLoader, self).load_arguments(command)
+        from azure.cli.command_modules.storage._params_v2 import load_arguments
+        load_arguments(self, command)
+
+
+class AzureStackStorageCommandsLoader(AzCommandsLoader):
```

Comment:
Naming is little bit tricky for me. Here are two command modules `storage.commands` and `storage.commands_v2`. Beside we have two command loader names `StorageCommandsLoader` and `AzureStackStorageCommandsLoader`. Is it possible to align these three names into two set of names? Why do we need such name `AzureStackStorage`?

Reply:
The naming of `AzureStack` requires a background knowledge of the internal requirement and dependencies. I agree with @MyronFanQiu that we should keep the name in sync.

Reply:
Better to keep loader name and name of command file and params file consistent for easier understand.

---
In reply to: [439186279](https://github.com/Azure/azure-cli/pull/13763#discussion_r439186279) [](ancestors = 439186279)

Reply:
done